examples/export_runs.py

```diff
@@ -0,0 +1,50 @@
+from domino import Domino
+import pandas as pd
+import json
+import os
+
+output_dir = "results"
+
+# connect to domino; be sure to have DOMINO_USER_API_KEY env set. Domino automatically includes this for you
+domino = Domino("nick/digits", api_key=os.environ['DOMINO_USER_API_KEY'], host="https://app.dominodatalab.com")
+raw_runs = domino.runs_list()['data']
+
+# print number of runs to STDOUT
+print "Details of {0} runs received".format(len(raw_runs))
+
+# create results directory if it doesn't exist
+if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
+
+# write API out to all_runs.json
+f = open('{0}/all_runs.json'.format(output_dir), 'w')
+f.write(json.dumps(raw_runs))
+
+# collect all runs into an array of dictionaries
+all_runs = []
+for run in raw_runs:
+    # flatten diagnosticStatistics which are stored in a nested array of dictionaries
+    if run['diagnosticStatistics'] != None:
+        for stat in run['diagnosticStatistics'].get('data',[]):
+            run['diagnosticStatistics.{0}'.format(stat['key'])] = stat['value']
```

Comment:
Storing `'diagnosticStatistics.{0}'.format(stat['key'])` in its own variable with a name would probably help readers of this code understand the intent a lot better.